src/index.ts

```diff
@@ -0,0 +1,135 @@
+export const queryByLabelText = bindFunction('queryByLabelText')
+export const queryAllByLabelText = bindFunction('queryAllByLabelText')
+export const findByLabelText = bindFunction('findByLabelText')
+export const findAllByLabelText = bindFunction('findAllByLabelText')
+export const getByPlaceholderText = bindFunction('getByPlaceholderText')
+export const getAllByPlaceholderText = bindFunction('getAllByPlaceholderText')
+export const queryByPlaceholderText = bindFunction('queryByPlaceholderText')
+export const queryAllByPlaceholderText = bindFunction(
+  'queryAllByPlaceholderText',
+)
+export const findByPlaceholderText = bindFunction('findByPlaceholderText')
+export const findAllByPlaceholderText = bindFunction('findAllByPlaceholderText')
+export const getByText = bindFunction('getByText')
+export const getAllByText = bindFunction('getAllByText')
+export const queryByText = bindFunction('queryByText')
+export const queryAllByText = bindFunction('queryAllByText')
+export const findByText = bindFunction('findByText')
+export const findAllByText = bindFunction('findAllByText')
+export const getByAltText = bindFunction('getByAltText')
+export const getAllByAltText = bindFunction('getAllByAltText')
+export const queryByAltText = bindFunction('queryByAltText')
+export const queryAllByAltText = bindFunction('queryAllByAltText')
+export const findByAltText = bindFunction('findByAltText')
+export const findAllByAltText = bindFunction('findAllByAltText')
+export const getByTitle = bindFunction('getByTitle')
+export const getAllByTitle = bindFunction('getAllByTitle')
+export const queryByTitle = bindFunction('queryByTitle')
+export const queryAllByTitle = bindFunction('queryAllByTitle')
+export const findByTitle = bindFunction('findByTitle')
+export const findAllByTitle = bindFunction('findAllByTitle')
+export const getByDisplayValue = bindFunction('getByDisplayValue')
+export const getAllByDisplayValue = bindFunction('getAllByDisplayValue')
+export const queryByDisplayValue = bindFunction('queryByDisplayValue')
+export const queryAllByDisplayValue = bindFunction('queryAllByDisplayValue')
+export const findByDisplayValue = bindFunction('findByDisplayValue')
+export const getByRole = bindFunction('getByRole')
+export const getAllByRole = bindFunction('getAllByRole')
+export const queryByRole = bindFunction('queryByRole')
+export const queryAllByRole = bindFunction('queryAllByRole')
+export const findByRole = bindFunction('findByRole')
+export const findAllByRole = bindFunction('findAllByRole')
+export const findAllByDisplayValue = bindFunction('findAllByDisplayValue')
+export const getByTestId = bindFunction('getByTestId')
+export const getAllByTestId = bindFunction('getAllByTestId')
+export const queryByTestId = bindFunction('queryByTestId')
+export const queryAllByTestId = bindFunction('queryAllByTestId')
+export const findByTestId = bindFunction('findByTestId')
+export const findAllByTestId = bindFunction('findAllByTestId')
```

Comment:
since we have to hardcode these now in TS, I added a test to ensure we keep parity with dom-testing-library.  hope you don't mind that I pushed to your branch. :)

Reply:
No problem 😉